samples/ck-rx65n/basic-sample/src/iotconnect_app.c

```diff
@@ -27,6 +28,16 @@ static char duid_buffer[IOTC_COMMON_NAME_MAX_LEN + 1]; // from ATECC608 common n
 #endif
 #define APP_VERSION "01.00.00"
 
+/***********************************************************************************************************************
+ * ThreadX object control blocks definitions
+ **********************************************************************************************************************/
+TX_THREAD               hs300x_sensor_thread;
+
+/***********************************************************************************************************************
+ * Stacks definitions
+ **********************************************************************************************************************/
```

Comment:
Minor Issue:
Blocks for section/function-header comments and similar reduce readability unless there is something that really needs to stand out. Please avoid using block secont commment sectrions in the future unless they are important. Especitally to separate variable/function types. You don't need to fix this now, but maybe remove it if you come around to editing this file.